libcudacxx/test/support/filesystem_dynamic_test_helper.py

```diff
@@ -1,22 +1,28 @@
 import sys
 import os
 import socket
-import stat
 
 # Ensure that this is being run on a specific platform
-assert sys.platform.startswith('linux') or sys.platform.startswith('darwin') \
-    or sys.platform.startswith('cygwin') or sys.platform.startswith('freebsd') \
-    or sys.platform.startswith('netbsd')
+assert (
```

Comment:
I'm also unsure if we should be mucking with libcu++'s testing scripts. 

@wmaxey

Reply:
If it is helpful in deciding, I have extremely high confidence in the correctness of the rewrites that `ruff` and other linters use. The number of invalid changes they make is vanishingly small. I looked at the changes in this file and a few others in `libcudacxx/`, they seem valid to me.

Reply:
I'll exclude this, too. We can work on this in a follow-on PR, as needed.)

Reply:
Please do! The formatting in those scripts is abysmal. :)

Reply:
OK, thanks! I'll work on that as soon as this PR is merged. It should be easy because I have the fixes already, although there was a weird "old" MSVC error that I'd need to debug if it still pops up.